admins/pageflow/entry.rb

```diff
@@ -49,7 +49,9 @@ module Pageflow
     end
 
     filter :title
-    filter :account
+    filter :account,
+           if: ->(_) { authorized?(:index, :accounts) },
+           collection: -> { Account.accessible_by(current_ability, :see_in_list) }
```

Comment:
I'm not aware of any restriction regarding `accessible_by` and actions that have a block. I think the block will simply be ignored and the scope returned by `AccountPolicy::Scope#resolve` will be used. The block is only used when the ability is inquired for individual records.

The fact that the block actually checks for manager permissions while the scope only looks for arbitrary membership, is in fact confusing, but is in the end caused by ActiveAdmin using `:read` to authorize `show` as well as `index`. But since the accounts displayed in the select should exactly be those displayed in the accounts index page, I think it's ok to mimic ActiveAdmin behavior here.